docs/site/content/posts/02-vmware-tanzu-community-edition-takes-off.md

```diff
@@ -0,0 +1,64 @@
+expand at an incredible rate. To date we've seen hundreds of thousands
+of web mentions and tens of thousands of visitors to our web site.
+Downloads are off the charts, and thousands of you have already begun
+exploring Tanzu Community Edition through hands-on workshops. Wow! What
+a great start for our community. 😊
+
+If you are new to the party, you can quickly catch up on Tanzu Community
+Edition release news and perspectives from a few of our favorite blog
+posts:
+
+- In VMware's [formal announcement](https://tanzu.vmware.com/content/blog/vmware-tanzu-community-edition-announcement) of Tanzu Community Edition, Amanda Katona breaks the news about the release. Take a quick read to catch up on what Tanzu Community Edition is, who it is for, and how it can be used.
+
+- Nigel Brown and Roger Klorese [welcome all to collaborate and contribute](https://tanzucommunityedition.io/posts/community-is-our-middle-name-welcome/) together in the community's inaugural blog.
+
+- Always insightful, [Craig McLuckie's blog post](https://tanzu.vmware.com/content/blog/vmware-tanzu-community-edition-kubernetes-cloud-native-ecosystem) discusses the importance of open source technologies in the cloud
+native landscape and the value of being able to easily consume them with Tanzu Community Edition.
+
+- Especially if you're a developer, you'll want to check out [Grant
+ Shipley's suggestions](https://tanzu.vmware.com/developer/blog/expanding-the-developer-toolbox-with-vmware-tanzu-community-edition/) for a few open source tools that you'll want to use with Tanzu Community Edition.
+
+- Dirk Hohndel offers perspective on the role that Tanzu Community Edition plays in expanding [VMware's commitment to and relationship with open source](https://blogs.vmware.com/opensource/2021/10/12/another-dimension-of-being-open/?utm_source=rss&utm_medium=rss&utm_campaign=another-dimension-of-being-open).
+
+Clearly, there is a real need for an offering like Tanzu Community
+Edition. That's surely no surprise: what application developer,
+infrastructure operator, site-reliability engineer, or DevOps engineer
+wouldn't want to learn or use Kubernetes today with a freely available,
+open source Kubernetes platform? One that can be installed and
+configured in minutes on a local workstation or favorite cloud, and
+that's backed by one of the world's most trusted enterprise technology
+providers? It all adds up to incredible opportunity for you: now is the
+right time to build your cloud native expertise, and this is just the
+tool to help you do that.
+
+It's easy to [download](https://tanzucommunityedition.io/download/) and
+install Tanzu Community Edition, and to find your path to success with
+like-minded colleagues and collaborators, lots of [training and
+education
+resources](https://tanzucommunityedition.io/resources/),
+and expert
+[support](https://tanzucommunityedition.io/community/)
+available through our community. [Join
+us](http://www.tanzucommunityedition.io) today!
```

Comment:
Make sure to link to https://tanzucommunityedition.io, instead of http, and www.